internal/common/enums.rs

```diff
@@ -233,6 +233,30 @@ macro_rules! for_each_enums {
                 Preserve,
             }
 
+            /// This enum specifies the horizontal alignment of the source image.
+            enum ImageHorizontalAlignment {
+                /// Aligns the source image at the center of the [`Image`](elements.md#image) element.
+                Center,
+                /// Aligns the source image at the left of the [`Image`](elements.md#image) element.
+                Left,
+                /// Aligns the source image at the right of the [`Image`](elements.md#image) element.
+                Right,
+                /// Aligns the source image at the start of the [`Image`](elements.md#image) element. This could be left or right depending on the language.
+                Start,
+                /// Aligns the source image at the end of the [`Image`](elements.md#image) element. This could be left or right depending on the language.
+                End,
```

Comment:
I suggest to leave out Start and End until we've decided on the actual layout mirroring mechanism. "depending on the language" is way too vague for me. If anything here should be link to a chapter that explains how Start and End can be influenced, which may or may not be triggered by translations, or perhaps an intermediate API, or by the locale.

So my vote is to omit it, but if Start/End shall remain, then the documentation needs to explain why somebody should use this (and that we may implement this in the future).

Reply:
The reason why I think we should have it is that I think people should write code now that should work when we support RTL layout.
But if you vote against it, i'll just remove it